client/my-sites/sidebar/sidebar.jsx

```diff
@@ -519,11 +519,11 @@ export class MySitesSidebar extends Component {
 		}
 
 		// Ignore Jetpack sites as they've opted into this interface.
-		if ( this.props.isJetpack ) {
+		if ( this.props.isJetpack && ! this.props.isAutomatedTransferSite ) {
```

Comment:
Completely optional but we can use a different name for the variable, if you get tired of typing:
```javascript
const { isJetpack, isAutomatedTransferSite: isAutomtedTransfer } = this.props;
if ( isJetpack && isAutomtedTransfer ) {
//... 
```